src/_P084_VEML6070.ino

```diff
@@ -0,0 +1,230 @@
+#ifdef USES_P084
+//#######################################################################################################
+//#################################### Plugin 084: VEML6070 UV ##########################################
+//#######################################################################################################
+
+// ESPEasy Plugin to for UV with chip VEML6070
+// written by Remco van Essen (https://github.com/RemCom)
+// Based on VEML6070 plugin from Sonoff-Tasmota (https://github.com/arendst/Sonoff-Tasmota)
+
+
+#define PLUGIN_084
+#define PLUGIN_ID_084         84
+#define PLUGIN_NAME_084       "UV - VEML6070 [TESTING]"
+#define PLUGIN_VALUENAME1_084 "UV-Raw"
+#define PLUGIN_VALUENAME2_084 "UV-Risk"
+#define PLUGIN_VALUENAME3_084 "UV-Power"
+
+#define VEML6070_ADDR_H             0x39           
+#define VEML6070_ADDR_L             0x38     
+#define VEML6070_RSET_DEFAULT       270000          // 270K default resistor value 270000 ohm, range from 220K..1Meg
+#define VEML6070_UV_MAX_INDEX       15              // normal 11, internal on weather laboratories and NASA it's 15 so far the sensor is linear
+#define VEML6070_UV_MAX_DEFAULT     11              // 11 = public default table values
+#define VEML6070_POWER_COEFFCIENT   0.025           // based on calculations from Karel Vanicek and reorder by hand
+#define VEML6070_TABLE_COEFFCIENT   32.86270591     // calculated by hand with help from a friend of mine, a professor which works in aero space things
+                                                    // (resistor, differences, power coefficients and official UV index calculations (LAT & LONG will be added later)
+#define D_UV_INDEX_1  "Low"      // = sun->fun
+#define D_UV_INDEX_2  "Mid"      // = sun->glases advised
+#define D_UV_INDEX_3  "High"     // = sun->glases a must
+#define D_UV_INDEX_4  "Danger"   // = sun->skin burns Level 1
+#define D_UV_INDEX_5  "BurnL1/2" // = sun->skin burns level 1..2
+#define D_UV_INDEX_6  "BurnL3"   // = sun->skin burns with level 3
+#define D_UV_INDEX_7  "OoR"      // = out of range or unknown
+
+#include <math.h>  
```

Comment:
Why is this include needed?

Reply:
Include was leftover from copied code. Now removed